docs/qa/docker.rst

```diff
@@ -0,0 +1,116 @@
+Docker for QA
+=============
+The following describes how to get a prebuilt instance of Normandy for
+testing. This is not suitable for development.
+
+Prerequisites
+-------------
+This guide assumes you have already installed and set up the following:
+
+1. Docker_
+2. `Postgres 9.4`_
+
+.. _Docker: https://docs.docker.com/engine/installation/
+.. _Postgres 9.4: http://www.postgresql.org/
+
+Preparing an environment file
+-----------------------------
+Normandy gets all of its configuration via environment variables. Docker
+has built in support for this pattern. For this set up, we will be using
+and environment file to specifiy this configuration, and having Docker
+read that file.
+
+The file can be named anything, but canonically is named ``.env``. The
+only thing that is required is a database config, which looks something
+like the below. We'll go over how to fill in the blanks next.
+
+.. code-block:: ini
+
+  DATABASE_URL=postgresql://<username>:<password>@<host>/<db_name>
+
+``<username>`` and ``<password>`` are the details to authenticate with
+Postgres. ``<db_name>`` is the name of the database to connect to, and
+is usually ``normandy``.
+
+``<host>`` is more complicated. This needs to be something that the
+Docker container can use to connect to Postgres. There are a few options here:
+
+Linux with local Postgres
+  Use ``172.17.0.1``
```

Comment:
If there are commands that can generate these IP addresses we should have them here (and maybe suggest the IP they probably are set to).